HelpSource/Classes/Float.schelp

```diff
@@ -65,3 +90,69 @@ pi.asStringPrec(6)
 7.4.asStringPrec(5)
 7.4.asStringPrec(50)
 ::
+
+
+log2(inf.asInteger)
+2 ** 31 - inf.asInteger
+
+subsection::Using Floats as replacement for Integers
+
+In SuperCollider, Floats are 64 bit wide. Because  an link::Classes/Integer:: is only 32 bit, it can capture only numbers up to code::± 2147483647::, which is code::2^31:: (one bit is for the sign), or about code::2 x 10^9::.
```

Comment:
the range is -2147483648 to +2147483647. On almost all consumer systems integers are represented in two's complement, which is not one bit for the sign. You might be thinking of floats which do have a sign bit in addition to exponent and mantissa bit groups.

Reply:
ah, good to know.